File: problems/python/448/Solution2.py
class Solution:
    def findDisappearedNumbers(self, nums: List[int]) -> List[int]:
        disappearedNums = set(range(1, len(nums) + 1))
        for num in nums:
        	if num in disappearedNums:
        		disappearedNums.remove(num)
        return disappearedNums

File: problems/kotlin/217/Solution2.kt
class Solution {
    fun containsDuplicate(nums: IntArray): Boolean {
        val uniqueNums = HashSet<Int>()
        for (num in nums) {
        	if (uniqueNums.contains(num)) return true
            uniqueNums.add(num)
        }
        return false
    }
}